### PR TITLE
Feature/story get sort

### PR DESCRIPTION
**Pull request by neha91192**
- Added changes to support fetching of stories in the following order: RECENT, MOST_UPVOTES, MOST_DOWNVOTES, OLDEST. Also, there is an optional query param - storyStatus which if not included in the request returns APPROVED stories.

- If you include storyStatus as "PENDING" or "DISAPPROVED" it will return stories accordingly. I need to add a check for admin access only in this case. 

- I'll also add a new API (once this is merged) which will return list of all the options available for Sorting - RECENT, MOST_UPVOTES, MOST_DOWNVOTES, OLDEST. 

- Once this PR is merged and used at front end for fetching approved/pending stories, we can safely remove /api/stories/approved and /api/stories/pending from StoryController

### CODE REVIEW

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesMockAdapter.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesMockAdapter.java]: Can we get rid of the mock adapter for story?

File: modules/application/src/main/java/edu/northeastern/truthtree/assembler/StoriesAssembler.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ public StoriesAssembler(ObjectMapper mapper) {
     } catch (IOException e) {
       e.printStackTrace();
     }
-    return storyDTO.getData();
+    return storyDTO.getData().getData();
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/assembler/StoriesAssembler.java]: I thought you renamed this to something else in previous pr? storyDTO variable isn't a `StoryDTO` obj in this case.

Comment (neha91192) [reply modules/application/src/main/java/edu/northeastern/truthtree/assembler/StoriesAssembler.java]: I remember I changed it. Let make changes and commit once again.

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesMockAdapter.java (hunk 1 above)
Comment (neha91192) [reply modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesMockAdapter.java]: Let's keep it for now?

**Pull request merged**